doc/classes/EditorPlugin.xml

```diff
@@ -108,6 +112,20 @@
 			<argument index="0" name="plugin" type="EditorInspectorPlugin">
 			</argument>
 			<description>
+				Registers a new inspector plugin. Inspector plugins are used to extend the default inspector. See [EditorInspectorPlugin] for more information.
+				[b]Note:[/b] Always use [method remove_inspector_plugin] to remove the [code]EditorInspectorPlugin[/code] if the [code]EditorPlugin[/code] is disabled.
```

Comment:
Weird use of `[code]` tags. Just use links.

```suggestion
				[b]Note:[/b] Always use [method remove_inspector_plugin] to remove the registered [EditorInspectorPlugin] when your [EditorPlugin] is disabled to prevent leaks and an unexpected behavior.
```